tests/services/permissions_test.py

```diff
@@ -0,0 +1,109 @@
+import uuid
+from datetime import datetime
+
+import pytest
+from fastapi_azure_auth.exceptions import InvalidAuth
+
+from laa_crime_application_store_app.models.application_schema import Application
+from laa_crime_application_store_app.services.permissions import (
+    validate_can_create,
+    validate_can_update,
+)
+
+
+def test_can_create_when_no_roles(normal_user):
```

Comment:
I don't think this test should be in here and maybe we feature flag this work instead.